about.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link rel="stylesheet" href="index.css">
+</head>
+
+<body>
+    <div class=" landing-page-container">
+        <div class="header-container">
+            <div class="logo">
+                <a href="index.html" title="Go to our Home Page">
+                    <img src="honeybeelogo.png" alt="Logo featuring a jolly honeybee holding a pot.">
+                </a>
+            </div>
+
+            <a class="base-frame red-frame" href="about.html" title="Learn about us">About</a>
+            <a class="base-frame yellow-frame" href="gallery.html" title="Check out pictures">Gallery</a>
+            <a class="base-frame green-frame" href="hours.html" title="Operating hours">Hours</a>
+            <a class="base-frame blue-frame" href="menu.html" title="Food menu for the kids">Menu</a>
+            <a class="base-frame violet-frame" href="contact.html" title="Our contact information">Contact</a>
+
+        </div>
+
+        <div class="page-seperator"></div>
+
+        <div class="middle-content">
+
+        </div>
+
+        <div class="page-seperator"></div>
+
+        <div class="bottom-content">
+            <div class="image">comb here</div>
+            <div class="footer-text">
+                Mon-Fri, 6:30 to 18:30 <br>
```

Comment:
```suggestion
                Mon-Fri, 6:30 AM to 6:30 PM<br>
```
Military time isn't too common here, let's keep it simple for this audience